api/checks/webhook.go

```diff
@@ -26,8 +26,7 @@ const (
 
 func isKnownAppID(appID int64) bool {
 	switch appID {
-	case wptfyiCheckAppID:
-	case checksStagingAppID:
+	case wptfyiCheckAppID, checksStagingAppID:
```

Comment:
@Hexcles @foolip @mdittmer 
A cautionary Golang tale is told in this little change. Fallthrough is not a thing, and `break` isn't needed.

    switch foo {
    case "bar", "baz":
        // "bar" or "baz" will do these lines
    case "qux":
    case "quux":
        // "quux" hits this line, but "qux" is actually an empty clause.